.github/actions/publish-to-npm/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish package to NPM
+
+description: "Publish new package version to the NPM registry"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        registry-url: 'https://registry.npmjs.org'
+
+    - name: Ensure clean install
+      run: yarn install --immutable
```

Comment:
This assumes that the repo uses yarn, should this always be the case?

Reply:
Yes, we might change it to something more agnostic, or if you know a tool for this ?